tests/jobs/scala/src/main/scala/KafkaCassandra.scala

```diff
@@ -0,0 +1,262 @@
+import java.util.{Date, Properties}
+
+import com.datastax.spark.connector._
+import com.datastax.spark.connector.cql.CassandraConnector
+import org.apache.kafka.clients.consumer.ConsumerConfig
+import org.apache.kafka.clients.producer.{KafkaProducer, ProducerConfig, ProducerRecord}
+import org.apache.kafka.common.serialization.StringDeserializer
+import org.apache.log4j.{Level, Logger}
+import org.apache.spark.internal.Logging
+import org.apache.spark.sql.SparkSession
+import org.apache.spark.streaming.kafka010.{ConsumerStrategies, KafkaUtils, LocationStrategies}
+import org.apache.spark.streaming.{Seconds, StreamingContext}
+
+
+
+/**
+  * A config class used to set the command line arguments.
+  */
+case class Config(
+                   appName: String = "KafkaRandomFeeder",
+                   brokers: String = "" ,
+                   topics: String = "",
+                   groupId: String = "TODO_USE_A_DIFFERENT_GROUP_ID",
+                   wordsPerSecond: Float = 1.0F,
+                   batchSizeSeconds: Long = 1L,
+                   cassandraKeyspace: String = "kafka_cassandra_stream",
+                   cassandraTable: String = "records",
+                   cassandra: Boolean = true,
+                   kerberized: Boolean = false,
+                   isLocal: Boolean = false
+                 )
+
+object KafkaRandomFeeder extends Logging {
+  def main(args: Array[String]): Unit = {
+    StreamingExamples.setStreamingLogLevels(Level.WARN)
+
+    val parser = StreamingExamples.getParser( this.getClass.getSimpleName, true ).parse(args, Config())
+    if (parser.isEmpty) {
+      logError("Bad arguments")
+      System.exit(1)
+    }
+
+    val config = parser.get
+    println(s"Using config: $config")
+
+    val spark = StreamingExamples.createSparkSession(config)
+
+    val streamingContext = new StreamingContext(spark.sparkContext, Seconds(config.batchSizeSeconds))
+
+    val wordsPerSecond = config.wordsPerSecond
+    val stream = streamingContext.receiverStream(new RandomWordReceiver(wordsPerSecond))
```

Comment:
since you only use it once, you may as well pass in `config.wordsPerSecond` like you do on L53